File: OrganizationService/OrganizationService.Service/ApiModels/Organization/UpdateOrganizationRequest.cs
using OrganizationService.Core.Constants;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OrganizationService.Service.ApiModels.Organization
{
    public class UpdateOrganizationRequest
    {
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public string? Address { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public Guid? Logo { get; set; }
        public string? LogoUrl { get; set; }
        public OrganizationStatus Status { get; set; }
    }
}

File: org-service/src/main/java/com/codecampus/organization/dto/request/UpdateOrganizationRequest.java
package com.codecampus.organization.dto.request;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@FieldDefaults(level = AccessLevel.PRIVATE)
@NoArgsConstructor
@AllArgsConstructor
public class UpdateOrganizationRequest {
  String description;
  String logoUrl;
  String email;
  String phone;
  String address;
  String status;
}